src/main/java/com/yahoo/bullet/rest/service/QueryService.java

```diff
@@ -30,8 +29,7 @@
 
 @Service
 public class QueryService {
-    // Exposed for testing only.
-    @Getter(AccessLevel.PACKAGE)
+    @Getter
```

Comment:
Don't expose runningQueries directly. Write a public method to return the number of current running queries. It will be easier to mock or change the implementation of QueryService.